README.md

```diff
@@ -255,6 +255,66 @@ This will cause React Native to import the `node_modules/bson/lib/bson.rn.cjs` b
 
 The `"exports"` definition in our `package.json` will result in BSON's CommonJS bundle being imported in a React Native project instead of the ES module bundle. Importing the CommonJS bundle is necessary because BSON's ES module bundle of BSON uses top-level await, which is not supported syntax in [React Native's runtime hermes](https://hermesengine.dev/).
 
+
+## BSON benchmarking
```

Comment:
I think this file is focused entirely on the bson library for users, the FAQ included, so I don't think we want to insert this between these sections. I would instead create a CONTRIBUTORS.md like we do in the node driver (you can lift half of that file for use here), and then add the benchmark instructions there (although in the driver we link to the centralized test directory).